src/OpenColorIO/transforms/FileTransform.cpp

```diff
@@ -985,10 +985,34 @@ OIIO_ADD_TEST(FileTransform, AllFormats)
     OIIO_CHECK_ASSERT(FormatExtensionFoundByName("spi3d", "spi3d"));
     OIIO_CHECK_ASSERT(FormatExtensionFoundByName("spimtx", "spimtx"));
     OIIO_CHECK_ASSERT(FormatExtensionFoundByName("vf", "nukevf"));
+}
+
+OIIO_ADD_TEST(FileTransform, FormatByIndex)
+{
+    OCIO::FormatRegistry & formatRegistry = OCIO::FormatRegistry::GetInstance();
 
+    for (unsigned int i = 0; i < formatRegistry.getNumRawFormats(); ++i)  {
```

Comment:
BTW, some Linux platforms also failed (i.e. Travis CI builds).

Reply:
Thanks @hodoulp, will fix this.

What I'm not sure is why the last commit I made (fd0223d5c7335003c39a6bcf8e4b012ccf4b18a3) was successful and not yours. Is it because of the rebase from master you made (that introduce new warnings as errors flags) ?